x/programs/runtime/runtime.go

```diff
@@ -53,6 +53,12 @@ func (r *WasmRuntime) Initialize(ctx context.Context, programBytes []byte, maxUn
 		return err
 	}
 
+	// enable bulk memory support only in testing/debug mode
+	// this is because some io operations from WASM require bulk memory support
+	if r.cfg.debugMode {
+		engineConfig.SetWasmBulkMemory(true)
+	}
+
```

Comment:
Good call, if we want to auto enable bulk memory we should add this directly to the config flag and document it so it is explicit.

https://github.com/ava-labs/hypersdk/blob/f735d9422ef7db029ec64bb26a2bf9af563dac98/x/programs/runtime/config.go#L206-L209